pkg/auth/handler/webapp/authflow_controller.go

```diff
@@ -1008,11 +1008,7 @@ func (c *AuthflowController) makeHTTPHandler(s *webapp.Session, screen *webapp.A
 		}
 
 		if err != nil {
-			if apierrors.IsAPIError(err) {
-				c.renderError(r.Context(), w, r, err)
-			} else {
-				panic(err)
-			}
+			c.renderError(r.Context(), w, r, err)
```

Comment:
The panic middleware will log the error for us. Now we do not panic, I think the error is not logged.

Reply:
You are right. Then I think the log should occur in error renderer.